draft-ietf-taps-transport-security.md

```diff
@@ -482,30 +485,33 @@ but this document considers them together, since that is the most common pattern
 
 #### IKEv2
 
-IKEv2 is a control protocol that runs on UDP port 500. Its primary goal is to generate keys for Security Associations (SAs).
-An SA contains shared (cryptographic) information used for establishing other SAs or keying ESP; See {{ESP}}.
-IKEv2 first uses a Diffie-Hellman key exchange to generate keys for the "IKE SA", which is a set of keys
-used to encrypt further IKEv2 messages. It then goes through a phase of authentication in which both peers
-present blobs signed by a shared secret or private key, after which another set of keys is derived, referred
-to as the "Child SA". These Child SA keys are used by ESP.
+IKEv2 is a control protocol that runs on UDP port 500 or TCP port 4500. (IKEv2 cannot run over
+UDP on any other port.) Its primary goal is to generate keys for Security Associations (SAs).
```

Comment:
Nit on this: I don't like that the entire sentence `(IKEv2 cannot run..` is a parenthetical. Also, while it technically isn't specified to listen on any other port, implementations often in practice use other ports on the initiator end, often because use of 500 or 4500 is restricted.

Reply:
Okay, noted. I'll just drop that sentence. It's sort of irrelevant anyway, which is why I made it a parenthetical.